frontend/src/pages/booking/components/BookingSummarySection/BookingSummarySection.tsx

```diff
@@ -1,12 +1,42 @@
 import { css } from '@emotion/react';
 import styled from '@emotion/styled';
 
+import { apiClient } from '../../../../common/apis/apiClient';
 import timeIcon from '../../../../common/assets/images/timeIcon.svg';
 import Button from '../../../../common/components/Button/Button';
 import TextWithIcon from '../../../../common/components/TextWithIcon/TextWithIcon';
+import { API_URL } from '../../../../common/constants/url';
 
-function BookingSummarySection() {
-  const handleBookingButtonClick = () => {};
+import type { BookingResponse } from '../../types/BookingResponse';
+
+interface BookingSummarySectionProps {
+  onBookingSuccess: (bookingResponse: BookingResponse) => void;
+  menteeName: string;
+  phoneNumber: string;
+  counselContent: string;
+}
+
+function BookingSummarySection({
+  onBookingSuccess,
+  menteeName,
+  phoneNumber,
+  counselContent,
+}: BookingSummarySectionProps) {
+  const handleClick = async () => {
+    try {
+      const response = await apiClient.post({
+        endpoint: `${API_URL.MENTORINGS}/1/reservation`,
```

Comment:
p2: endpoint에 숫자 1이 고정되어 있어서 바뀌어야 할 것 같아 👍

```suggestion
        endpoint: `${API_URL.MENTORINGS}/${id}/reservation`,
```